targets/javascript/templates/PlayFab_Api.js.ejs

```diff
@@ -152,6 +152,79 @@ if(!PlayFab._internalSettings) {
             }
 
             xhr.send(requestBody);
+        },
+
+        authenticationContext: {
+            PlayFabId: null,
+            EntityId: null,
+            EntityType: null,
+            SessionTicket: null,
+            EntityToken: null
+        },
+
+        UpdateAuthenticationContext: function (authenticationContext, result) {
+            var authenticationContextUpdates = {};
+            if(result.data.PlayFabId !== null) {
+                PlayFab._internalSettings.authenticationContext.PlayFabId = result.data.PlayFabId;
+                authenticationContextUpdates.PlayFabId = result.data.PlayFabId;
+            }
+            if(result.data.SessionTicket !== null) {
+                PlayFab._internalSettings.authenticationContext.SessionTicket = result.data.SessionTicket;
+                authenticationContextUpdates.SessionTicket = result.data.SessionTicket;
+            }
+            if (result.data.EntityToken !== null) {
+                PlayFab._internalSettings.authenticationContext.EntityId = result.data.EntityToken.Entity.Id;
+                authenticationContextUpdates.EntityId = result.data.EntityToken.Entity.Id;
+                PlayFab._internalSettings.authenticationContext.EntityType = result.data.EntityToken.Entity.Type;
+                authenticationContextUpdates.EntityType = result.data.EntityToken.Entity.Type;
+                PlayFab._internalSettings.authenticationContext.EntityToken = result.data.EntityToken.EntityToken;
+                authenticationContextUpdates.EntityToken = result.data.EntityToken.EntityToken;
+            }
+            // Update the authenticationContext with values from the result
+            authenticationContext = Object.assign(authenticationContext, authenticationContextUpdates);
+            return authenticationContext;
+        },
+
+        AuthInfoMap: {
+            "X-EntityToken": {
+                authAttr: "entityToken",
+                authError: "errorEntityToken"
+            },
+            "X-Authorization": {
+                authAttr: "sessionTicket",
+                authError: "errorLoggedIn"
+            },
+            "X-SecretKey": {
+                authAttr: "developerSecretKey",
+                authError: "errorSecretKey"
+            }
+        },
+
+        GetAuthInfo: function (request, authKey) {
+            // Use the most-recently saved sessionTicket, unless one was provided in the request via the PlayFabId property
+            var authAttr = PlayFab._internalSettings.AuthInfoMap[authKey].authError;
+            var authError = PlayFab._internalSettings[authAttr];
+            authAttr = PlayFab._internalSettings.AuthInfoMap[authKey].authAttr;
```

Comment:
And now you're re-assigning authAttr, so I think you should figure out the correct two lines out of these three lines. It seems like authError is probably being set wrong, and the first authAttr assignment should be deleted